src/odyssey/java/assignments/operators/DemoClass.java

```diff
@@ -0,0 +1,21 @@
+package src.odyssey.java.assignments.operators;
+
+import java.util.Date;
+
+public class DemoClass {
+    public static void main(String[] args) {
+//        BankAccount objAccount = new BankAccount("123-45-6789","Khaled Hassan");
+////        objAccount.accountHolderName = "Khaled Hassan";
+//        objAccount.printStatement();
+//
+//        BankAccount objNaimAccount = new BankAccount();
+//        objNaimAccount.accountHolderName = "Naim Islam";
+//        objNaimAccount.printStatement();
+//
```

Comment:
When you push your changes make sure to delete the commented code as well. It will make the code nicer.